modules/social_features/social_follow_taxonomy/src/SocialFollowTaxonomyInstallHelper.php

```diff
@@ -0,0 +1,131 @@
+    $this->configFactory = $config_factory;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container): self {
+    return new static(
+      $container->get('config.factory'),
+    );
+  }
+
+  /**
+   * Install required configuration.
+   */
+  public function installConfiguration(): void {
+    // Install views filter configuration.
+    $config = $this->configFactory->getEditable('views.view.user_admin_people');
+    if (!empty($config->get('display.default.display_options.filters'))) {
+      $config->set('display.default.display_options.filters.social_follow_taxonomy_follow_filter', $this->getFollowTaxonomyFilterConfiguration());
+      $config->save();
+    }
+  }
+
+  /**
+   * Uninstall required configuration.
+   */
+  public function uninstallConfiguration(): void {
+    // Uninstall views filter configuration.
+    $config = $this->configFactory->getEditable('views.view.user_admin_people');
+    $config->clear('display.default.display_options.filters.social_follow_taxonomy_follow_filter');
+    $config->save();
+  }
+
+  /**
+   * Get "FollowTaxonomyViewsFilter" views filter configuration.
+   */
+  private function getFollowTaxonomyFilterConfiguration(): array {
+    return [
+      'id' => 'social_follow_taxonomy_follow_filter',
+      'table' => 'users_field_data',
+      'field' => 'social_follow_taxonomy_follow_filter',
+      'relationship' => 'none',
+      'group_type' => 'group',
+      'admin_label' => '',
+      'operator' => 'or',
+      'value' => [],
+      'group' => 1,
+      'exposed' => TRUE,
+      'expose' => [
+        'operator_id' => 'social_follow_taxonomy_follow_filter_op',
+        'label' => 'Following by content tags',
+        'use_operator' => FALSE,
+        'operator' => 'social_follow_taxonomy_follow_filter_op',
+        'operator_limit_selection' => FALSE,
+        'operator_list' => [],
+        'identifier' => 'social_follow_taxonomy_follow_filter',
+        'required' => FALSE,
+        'remember' => FALSE,
+        'multiple' => TRUE,
+        'remember_roles' => [
+          'authenticated' => 'authenticated',
+        ],
+        'reduce' => FALSE,
+      ],
+      'is_grouped' => FALSE,
+      'group_info' => [
+        'label' => '',
+        'description' => '',
+        'identifier' => '',
+        'optional' => TRUE,
+        'widget' => 'select',
+        'multiple' => FALSE,
+        'remember' => FALSE,
+        'default_group' => 'All',
+        'default_group_multiple' => [],
+        'group_items' => [],
+      ],
+      'reduce_duplicates' => FALSE,
+      'type' => 'select',
+      'limit' => TRUE,
+      'vid' => [
+        'social_tagging' => 'social_tagging',
+      ],
+      'hierarchy' => FALSE,
+      'error_message' => TRUE,
+      'entity_type' => 'user',
+      'plugin_id' => 'social_follow_taxonomy_follow_filter',
+    ];
+  }
+
+  /**
+   * Implements hook_update_11001().
+   */
+  public function update11001(): void {
```

Comment:
Maybe index will be 11002, I have added milestone 11.2 to this PR

Reply:
Fixed.